src/search_renderer.cpp

```diff
@@ -86,6 +86,22 @@ void SearchRenderer::setSearchProtocolPrefix(const std::string& prefix)
   this->searchProtocolPrefix = prefix;
 }
 
+kainjow::mustache::data buildQueryData
+(
+  const std::string& pattern,
+  const std::set<std::string>& bookIds
+) {
+  kainjow::mustache::data query;
+  query.set("pattern", kiwix::encodeDiples(pattern));
+  std::ostringstream ss;
+  ss << "?pattern=" << urlEncode(pattern);
+  for (auto& bookId: bookIds) {
+    ss << "&books.id="<<urlEncode(bookId);
+  }
+  query.set("path", ss.str());
```

Comment:
`path` isn't a good name for this attribute of the query. What about getting rid of `searchProtocolPrefix` in the mustache template and replacing the stable combination `{{searchProtocolPrefix}}{{query.path}}` with `{{unpaginatedQuery}}`?

Reply:
Done